dashboards/ecs/ecs_instance_report_age.sp

```diff
@@ -128,8 +128,8 @@ query "ecs_instance_1_year_count" {
 query "ecs_instance_age_table" {
   sql = <<-EOQ
     select
-      i.instance_id as "Instance ID",
       i.tags ->> 'Name' as "Name",
+      i.instance_id as "Instance ID",
       now()::date - i.start_time::date as "Age in Days",
```

Comment:
I think creation_time is the correct column reference to determine the age of the ECS instance.

@madhushreeray30 could you please update the query to use the new column?